lib/term_glyph.dart

```diff
@@ -8,5 +8,10 @@ import 'src/generated.dart';
 
 /// Returns [glyph] if Unicode glyph are allowed, and [alternative] if they
 /// aren't.
+@Deprecated('Use glyphOrFallback instead. Will be removed in 2.0.0')
```

Comment:
According to the documentation for `@Deprecated`, the argument is only supposed to include the version where it'll be removed. The alternative should be listed in the main documentation.